Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -48,14 +48,22 @@ - (void)testInvalidDsn {
     XCTAssertEqual(kSentryErrorInvalidDsnError, error.code);
     XCTAssertNil(options);
 }
-    
+
 - (void)testRelease {
-    SentryOptions *options = [self getValidOptions:@{}];
-    XCTAssertNil(options.releaseName);
-    
-    options = [self getValidOptions:@{@"release": @"abc"}];
+    SentryOptions *options = [self getValidOptions:@{@"release": @"abc"}];
     XCTAssertEqualObjects(options.releaseName, @"abc");
 }
+
+- (void)testNoReleaseSetUsesDefault {
+    SentryOptions *options = [self getValidOptions:@{}];
+
+    NSDictionary *infoDict = [[NSBundle mainBundle] infoDictionary];
+    NSString *expectedRelease = [NSString stringWithFormat:@"%@@%@+%@",
+                                                           infoDict[@"CFBundleIdentifier"],
+                                                           infoDict[@"CFBundleShortVersionString"],
+                                                           infoDict[@"CFBundleVersion"]];
```

Comment:
Nitpick
```suggestion
                                 infoDict[@"CFBundleIdentifier"],
                                 infoDict[@"CFBundleShortVersionString"],
                                 infoDict[@"CFBundleVersion"]];

```

Reply:
@philipphofmann use the logaf scale instead of `Nitpick`